pkg/tss/keys.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package tss
+
+import (
+	"bytes"
+	"crypto/sha1"
+	"crypto/sha256"
+	"fmt"
+	"io"
+
+	"github.com/google/go-tpm-tools/proto"
```

Comment:
This proto will add quite some binary size. Are you sure you want it?

Reply:
That's fine